src/application/variables/assignVarFromValue.ts

```diff
@@ -51,13 +51,8 @@ export const assignVarFromValue = (
       casing: globals?.variableCasing
     }
   })
-  const varValue =
-    varSetting?.value && hasTpl(varSetting.value)
-      ? generatedValue
-      : typeof varSetting.value === 'string'
-      ? `"${varSetting.value}"`
-      : varSetting.value
-  // const varValue = typeof varSetting.value === 'string' ? `"${varSetting.value}"` : varSetting.value
+  const convVal = varSetting?.value && hasTpl(varSetting.value) ? generatedValue : varSetting.value
+  const varValue = typeof convVal === 'string' ? `"${convVal}"` : convVal
```

Comment:
is a style thing, but just don't like not knowing what `convVal` refers too when you see it in isolation. We're not gaining by abbreviating variables name....

Reply:
Good comment, it was a temp variable, but even that should be descriptive enough.

Changed it to:
```
  const processedValue = varSetting?.value && hasTpl(varSetting.value) ? generatedValue : varSetting.value
  const varValue = typeof processedValue === 'string' ? `"${processedValue}"` : processedValue
```

Is that better?